lib/DBDefs/Default.pm

```diff
@@ -472,6 +472,9 @@ sub SOLRCLOUD_RSYNC_BANDWIDTH { undef }
 sub SOLRCLOUD_SSH_CIPHER_SPEC { undef }
 sub SEARCH_INDEXES_DUMP_COMPRESSION_LEVEL { undef }
 
+# Used to compress dump using xz or zstd.
```

Comment:
Maybe something slightly more specific like this:
```suggestion
# Controls the number of threads to use when compressing JSON dumps or search
# index dumps. Passed as `-T` or `--threads` to xz and zstd.
```

Reply:
Thanks, applied it.